src/sections/LoginAuth0.js

```diff
@@ -23,38 +23,42 @@ class LoginAuth0 extends Component {
       domain: props.domain,
       clientID: props.clientId
     });
+    this.state = {
+      isLoggingIn: false
+    };
   }
 
   componentDidMount() {
-    const props = this.props;
     this.webAuth.parseHash(
       { hash: window.location.hash },
       (err, authResult) => {
+        window.location.hash = '';
         if (err) {
-          // TODO: Handle errors
+          // TODO: Handle errors better
+          alert(`${err.error}: ${err.errorDescription}`);
           return console.error(err);
         }
         if (!authResult || !authResult.accessToken) {
           return;
         }
-        window.location.hash = '';
+        this.setState({ isLoggingIn: true });
```

Comment:
Consider using the [callback](https://reactjs.org/docs/react-component.html#setstate) on `setState`, instead of passing in a hash.  I think this way of assigning state is old practice.

(Same on line 61 below.)

Reply:
@object88 The callback seems like an overkill if we are updating a single value. I think perfectly fine to use a hash in this case.

Reply:
I believe that only matters for state updates that are dependent on the current state since `setState` is asynchronous. 
> "If the next state depends on the previous state, we recommend using the updater function form, instead"

-- https://reactjs.org/docs/react-component.html#setstate

However, it might be best to just be consistent everywhere. That way we encourage people to use functions which work in all cases instead of objects which sometimes might not work properly. I'll make the change

Reply:
Actually, I might just push those changes out into a separate piece of work since there are several other cases of `setState` using objects, and if I merge this as is, I don't have to re-test things (which is all manual right now since there are essentially no automated tests for this project currently).